routes/webhooks/sffd.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express');
+const HttpStatus = require('http-status-codes');
+
+const models = require('../../models');
+
+const router = express.Router();
+
+const UNIT_SFFD_REGEX = /^\d+$/;
+const UNIT_AM_REGEX = /^AM\d+$/;
+const UNIT_KM_REGEX = /^KM\d+$/;
+
+router.post('/cad', async (req, res) => {
```

Comment:
There is currently no authentication on this endpoint. Anyone could POST appropriately formatted data to it and create fake records.

Any thoughts on how to protect this endpoint? IP whitelisting from the SFFD? Giving them some a "secret" key to put in the POST header, tied to a BatsUser account?

Reply:
Doesn't seem like it would be too hard to add another Passport strategy for bearer auth tokens: http://www.passportjs.org/packages/passport-http-bearer/. I'm guessing we'd want a new table for the auth tokens too.

Do you think other EMS providers would be able to do this as well?

Reply:
Actually, we could probably just stick the auth token in the batsuser table to make life easier.